test/test_coordinates_space.py

```diff
@@ -0,0 +1,358 @@
+# Copyright (C) 2023 Shichao Wu, Alex Nitz
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+import numpy
+from pycbc.coordinates import space
+import unittest
+from utils import simple_exit
+
+
+seed = 8202
+numpy.random.seed(seed)
+
+# ranges to draw random numbers for each parameter
+RANGES = {
+    "tc_geo" : (1126259462.43, 1426259462.43),
```

Comment:
This part randomly generates parameters set from those ranges, to see if my transform functions are reversible or not because LISA's orbit is periodic, so it's OK to choose a random value. Another reason is that this range covers the GPS time when 3G detectors will be online if I remember correctly.